src/Lidar.cc

```diff
@@ -245,7 +253,7 @@ bool Lidar::PublishLidarScan(const std::chrono::steady_clock::duration &_now)
   // keeping here the sensor name instead of frame_id because the visualizeLidar
   // plugin relies on this value to get the position of the lidar.
   // the ros_ign plugin is using the laserscan.proto 'frame' field
-  frame->add_value(this->Name());
+  frame->add_value(this->FrameId());
```

Comment:
I just saw this in passing. It seems like the original author left a comment about using `Name` here instead of `frame_id`. Can you verify if this is still an issue with VisualizeLidar?

Reply:
found that lidar visualization is broken when `<gz_frame_id>` is specified. I think the visualization plugin should not always assume that the frame id is the name so created a fix in https://github.com/gazebosim/gz-sim/pull/2481.

Reply:
I also removed the comment above.

Reply:
backported lidar visualization fix (https://github.com/gazebosim/gz-sim/pull/2481) to fortress in https://github.com/gazebosim/gz-sim/pull/2483.